evap/evaluation/models.py

```diff
@@ -512,19 +514,39 @@ class TextAnswer(Answer):
     """A free-form text answer to a question (usually a comment about a course
     or a contributor)."""
 
-    elements_per_page = 5
-
     reviewed_answer = models.TextField(verbose_name=_("reviewed answer"), blank=True, null=True)
     original_answer = models.TextField(verbose_name=_("original answer"), blank=True)
-
-    checked = models.BooleanField(verbose_name=_("answer checked"), default=False)
-    hidden = models.BooleanField(verbose_name=_("hide answer"), default=False)
+    
+    HIDDEN = 'hidden'
+    PUBLISHED = 'published'
+    PRIVATE = 'private'
+    NOT_REVIEWED = 'not_reviewed'
+    TEXT_ANSWER_STATES = (
+        (HIDDEN, _('hidden')),
+        (PUBLISHED, _('published')),
+        (PRIVATE, _('private')),
+        (NOT_REVIEWED, _('not reviewed')),
```

Comment:
come on, just because you two wanted it i renamed them to this. i don't think two letters are better, so i will not change it again.